.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test and Build
```

Comment:
Since this workflow is only testing I think it's probably better to just call it "Test"